package.json

```diff
@@ -95,6 +95,7 @@
     "@swc/helpers": "0.5.5",
     "@testing-library/jest-dom": "6.1.2",
     "@testing-library/react": "13.0.0",
+    "@types/busboy": "1.5.3",
```

Comment:
We didn't have the types here before, this isn't bundled, but enhances the code that uses `busboy` with some types.